lib/project_types/extension/messages/messages.rb

```diff
@@ -139,6 +139,10 @@ module Messages
               Usage: {{command:%1$s extension tunnel status}}
         HELP
       },
+      check: {
+        help: "Check your extension for errors, suggestions, and best practices.",
```

Comment:
We don't autocorrect offences, do we?

Reply:
🤔 

Passing `-a` should fix offenses that are fixable. I think there are only a few checks that support this right now.

Reply:
Yeah just some autocorrect for `SpacesInsideBraces` & `ConvertIncludeToRender` for now. @hannakebedom is working on more!